src/components/qr/use-resize-handle-props.property.spec.ts

```
@@ -0,0 +1,186 @@
+import { describe, it, expect, vi } from "vitest";
+import fc from "fast-check";
+import { renderHook, act } from "@testing-library/react";
+
+vi.mock("@react-aria/interactions", () => {
+  return {
+    useMove: (handlers: any) => ({
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)